### DISCUSSION TITLE
Consulta instalación compilador

### DISCUSSION THREAD

**Discussion by Clirussa** [INGCOM-UNRN-P1/cursada-2023#34] [Preguntas y respuestas] [answered]
Hola, me aparece esto, ¿Qué falta?
![P1-problemagcc](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/102642611/af6faf6f-ce6a-4c48-82a4-5e6365a8c6f7)

**Comment by Clirussa** [comment]
Según lo que estuve viendo, es que me falta el gcc pero no entiendo a que se refiere con que "enviar ese directorio a la raíz del disco C, de forma que la ruta quede como C:\mingw64". Según lo que aparece en el wiki.

**Reply by JeanFranco99** [reply, marked as answer]
Cuando descomprimis el .rar, te queda una carpeta `mingw64` (En donde hayas seleccionado que se descomprima con la opcion de __Extraer aqui__)
Haces click derecho en la carpeta `mingw64` -> mover a la carpeta -> Disco Local (C:) y apretas Mover
![image](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/83616156/cf860aeb-e0e3-43b9-8da5-00ee7706a995)
![image](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/83616156/67996a6d-8c73-4558-8a38-3b80df677da0)


Despues tenes que hacer lo que te aparece en la wiki para agregarla al path y que funcione con cualquier consola

**Reply by martinvilu** [reply]
> Despues tenes que hacer lo que te aparece en la wiki para agregarla al path y que funcione con cualquier consola

https://github.com/INGCOM-UNRN-P1/cursada-2023/wiki/Instalaci%C3%B3n-de-las-herramientas

Edit: agregue el paso como está descrito acá en el wiki para futura referencia.
¡Punto para @JeanFranco99 !

**Reply by Clirussa** [reply]
Buenas, lo intenté hacer así pero no me aparece la opción "mover" por lo que copié y pegué la carpeta en el C:. ¿Es lo mismo? Después edité las variables de entorno y le di la ubicación de ese archivo. Pero sigue sin reconocerme el gcc. Probé varias veces y estuve usando la consola del Git para ver si había algún problema con eso, pero eso me funciona. No se que más hacer. Espero sugerencias. Gracias

**Reply by martinvilu** [reply]
Botón derecho en lo que se descompacta, (`mingw64`), clic en “Cortar”, vas a “Mi PC”, entras en el “Disco C”, sobre un espacio en blanco del explorador de archivos, “Pegar”.

Si los otros pasos estaban completados como está indicado en el wiki (lo del "path"), debiera de funcionar.

**Reply by Clirussa** [reply]
Al fin logré que funcione todo. Me faltaba modificar bien las variables de entorno.
